### ISSUE TITLE
Automate sending welcome email to user upon sign up

### ISSUE THREAD

**Issue by tdowds** [tdowds/rememberthee-website#18]
When a user signs up, I would like to automatically send them an email welcoming them and telling them what their next steps are.

Required issues to be completed before this one:
- https://github.com/tdowds/rememberthee-website/issues/17

Acceptance Criteria:

- [ ] in `templates` create an email template
- [ ] in the `POST /sign-up` route, send email template to the email given in addition to the email sent to `trevor@rememberthee.com`

**Comment by michaelyaacoub** [comment]
Sounds good! I believe I could get this done tomorrow.

**Comment by michaelyaacoub** [comment]
What do we want the automated email says back?

**Comment by tdowds** [comment]
```
Hi {firstName},

Thanks for filling out the form on www.rememberthee.com. As the website mentioned, the app is still in the works, and I expect to have a date to share for when it will be available in the next few weeks.

In the meantime, I would love to hear how you heard about Remember Thee and what you're most interested in using it for. Feel free to respond directly to me at this email if you're interested in sharing more or if you have any questions.

If you'd like to share any additional thoughts or feedback and walk through a prototype of the app, I'd be happy to hop on a zoom call or meet in person if that's something that would interest you.

Let me know, and thanks again for reaching out.

Trevor Dowds, Founder and Creator
```
please also place this image below my name in the sign off:
![image](https://user-images.githubusercontent.com/5726560/140791815-15857db7-d537-4435-9ca3-3c45fe6dbcb6.png)

**Comment by michaelyaacoub** [comment]
I'm still figuring out how to achieve this in Flask, it would be great if you have resources I could look at.

What I have done so far is:




`fName = ['firstName']	
message = "Thanks for filling out the form on www.rememberthee.com. As the website mentioned, the app is still in the works, and I expect to have a date to share for when it will be available in the next few weeks."

server = smtplib.SMTP("smtp-mail.outlook.com", 587)
server.login("email", "passowrd")
server.sendmail("trevor@rememberthee.com", fName, message)`
> for login password will look for an Environmental variable to save the password secured! os.getenv("PASSWORD")

**Comment by tdowds** [comment]
closed via #24
